test/bigtest/network/config.js

```diff
@@ -613,4 +613,29 @@ export default function config() {
       'helpText': 'help text',
     }]
   });
+
+  this.post('/credentials', (schema, { requestBody }) => {
+    const creds = JSON.parse(requestBody);
+    return server.create('credential', creds);
+  });
+
+  this.put('/credentials/:id', ({ users }, request) => {
+    return users.find(request.params.id).attrs;
+  });
+
+  this.get('/credentials/:id', ({ users }, request) => {
+    return users.find(request.params.id).attrs;
+  });
+
+  // this.get('credentials/:userId', {
```

Comment:
Should this be removed?

Reply:
Sure, definitely. I still continue to work on tests.